tests/playwright-test/playwright-test-fixtures.ts

```diff
@@ -255,6 +255,10 @@ export function stripAnsi(str: string): string {
   return str.replace(asciiRegex, '');
 }
 
+export function trimLines(text: string): string {
```

Comment:
this is more like `trimEndLines`

If it does trim on both sides though, then you'd have a nice indent-aligned text expectations below!